WalletWasabi.Gui/Controls/SuggestLabelViewModel.cs

```diff
@@ -51,6 +53,8 @@ public ObservableCollection<SuggestionViewModel> Suggestions
 			set => this.RaiseAndSetIfChanged(ref _suggestions, value);
 		}
 
+		public WalletService WalletService { get; }
```

Comment:
Curious CodeFactor didn pick this up, Properties should come before methods I think is the rule.

Reply:
Also should be private.

Reply:
> @yahiheb Curious CodeFactor didn pick this up, Properties should come before methods I think is the rule.

That is correct.
I don't know how CodeFactor is working, sometimes it picks things up (like in your [PR](#3116)) sometimes it doesn't.
You can take a look at these PRs where we tested that rule: #3043, #3044, #3057

Reply:
never mind, I'll fix it later after we get the multi wallet done, if it doesnt get mopped up later.

Reply:
According to my theory: if one of the CodeFactor rules are violated and merged into the master then that rule will be skipped. 
I agree, fix codefactor issues after multiwallet.

Reply:
Sometimes it is skipped sometimes it is not.